packages/block-editor/src/store/defaults.js

```diff
@@ -152,6 +153,7 @@ export const SETTINGS_DEFAULTS = {
 	__experimentalEnableLegacyWidgetBlock: false,
 	__experimentalEnableMenuBlock: false,
 	__experimentalBlockDirectory: false,
+	__experimentalEnableFullSiteEditing: false,
```

Comment:
Since I've started to see this popping up in [other PRs](https://github.com/WordPress/gutenberg/pull/18029), what do you think if we add it in its own PR to make it easier to contribute towards this feature?

Reply:
It's already been merged. This is just adding missing default values and JSDoc comments.